hack/csi_client/pkg/csi/client.go

```diff
@@ -24,9 +24,11 @@ const (
 
 var (
 	goOs          = runtime.GOOS
+	errVolumeAttr = fmt.Errorf("volume attributes are required")
 	errHost       = fmt.Errorf("host name is required")
```

Comment:
Are all of these errs still needed?

Reply:
No they are not, I forgot to remove them.